packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -1,32 +1,18 @@
 const GroupHelper = require('../utils/GroupHelper');
-const logger = require('../utils/logger');
 /**
  * StatsGroup gathers information about the stats options
  */
 class StatsGroup extends GroupHelper {
-    static validOptions() {
-        return ['none', 'errors-only', 'minimal', 'normal', 'detailed', 'verbose', 'errors-warnings', true, false];
-    }
-
     constructor(options) {
         super(options);
     }
 
     resolveOptions() {
-        if (this.args.verbose && this.args.stats) {
-            logger.warn('Conflict between "verbose" and "stats" options. Using verbose.');
-            this.opts.options.stats = 'verbose';
-        } else {
-            if (this.args.verbose) {
-                this.opts.options.stats = 'verbose';
-            } else if (this.args.stats && !StatsGroup.validOptions().includes(this.args.stats)) {
-                logger.warn(`'${this.args.stats}' is invalid value for stats. Using 'normal' option for stats`);
-                this.opts.options.stats = 'normal';
-            } else {
-                this.opts.options.stats = this.args.stats;
-            }
+        const { stats, json } = this.args;
+        if (typeof stats !== undefined) {
```

Comment:
Shouldn't it be like this?

```suggestion
        if (typeof stats !== 'undefined') {
```

Reply:
nope not a string

Reply:
simpler would be to check `stats !== undefined`